test/check-storage

```diff
@@ -734,147 +734,147 @@ class TestStorage(MachineCase):
-        b.click("#format-format")
-        b.wait_popdown("storage_format_dialog")
-        b.wait_popup("error-popup")
-        b.wait_in_text("#error-popup-message", "is in use: mounted at")
-        b.click("#error-popup button")
-        b.wait_popdown("error-popup")
-
-        self.content_action(1, "Delete")
-        self.confirm()
-        b.wait_popup("error-popup")
-        b.wait_in_text("#error-popup-message", "is in use: mounted at")
-        b.click("#error-popup button")
-        b.wait_popdown("error-popup")
-
-        b.click('#drive_format')
-        b.wait_popup("storage_format_disk_dialog")
-        b.click("#format-disk-format")
-        b.wait_popdown("storage_format_disk_dialog")
-        b.wait_popup("error-popup")
-        b.wait_in_text("#error-popup-message", "is in use: mounted at")
-        b.click("#error-popup button")
-        b.wait_popdown("error-popup")
-
-        # Unmount and format the disk.
-
-        m.execute("umount /mnt")
-        b.wait_in_text("#storage_detail_content", "not mounted")
-
-        b.click('#drive_format')
-        b.wait_popup("storage_format_disk_dialog")
-        b.click("#format-disk-format")
-        b.wait_popdown("storage_format_disk_dialog")
-
-        b.wait_in_text("#storage_detail_content", "50 MB Free Space")
-
-    def testHidden(self):
-        m = self.machine
-        b = self.browser
-
-        self.login_and_go("storage", href="/storage/devices")
-
-        self.add_disk("50M", serial="DISK1")
-        b.wait_in_text("#storage_drives", "DISK1")
-
-        # Create a volume group with a logical volume with a encrypted
-        # filesystem.
-
-        m.execute("vgcreate TEST /dev/disk/by-id/scsi-0QEMU_QEMU_HARDDISK_DISK1")
-        m.execute("lvcreate TEST -n lvol -L 48")
-        m.execute("echo einszweidrei | cryptsetup luksFormat /dev/TEST/lvol")
-        m.execute("echo einszweidrei | cryptsetup luksOpen /dev/TEST/lvol dm-test")
-        m.execute("udevadm settle")
-        m.execute("mke2fs -q -L FS /dev/mapper/dm-test")
-        m.execute("echo '/dev/mapper/dm-test /mount ext4 defaults 0 0' >>/etc/fstab")
-        b.wait_dbus_prop("com.redhat.Cockpit.Storage.Block", "MountPoint", "/mount")
-
-        # Now doubly hide /dev/mapper/dm-test by locking dm-test and deactivating /dev/TEST/lvol
-        m.execute("cryptsetup luksClose dm-test")
-        m.execute("lvchange TEST/lvol -a n")
-
-        # Restart cockpit-wrapper by logging in again
-        self.allow_restart_journal_messages()
-        b.relogin('storage')
-
-        # Delete the volume group and check that fstab has been cleaned up
-        b.wait_in_text("#storage_vgs", "TEST")
-        b.click('tr:contains("TEST")')
-        b.click_action_btn('#vg_action_btn', "Delete")
-        self.confirm()
-        b.enter_page("storage")
-        check_eq(m.execute("grep /mount /etc/fstab || true"), "")
+        self.assertEqual(m.execute("grep /thin /etc/fstab || true"), "")
+
+        def testResize(self):
```

Comment:
did you increase this indentation on purpose?
if so, why disable the tests in this manner? `check-verify` still refers to them